src/components/App/App.js

```diff
@@ -1,80 +1,29 @@
-import React, {useState, useEffect} from 'react';
-import ItemList from '../ItemList/ItemList';
-import InputItem from '../InputItem/InputItem';
-import Footer from '../Footer/Footer';
-import styles from './App.module.css';
-
-
-const App = () => {
-	const initialState = {
-		items: [
-			{
-				value: 'Покормить кота',
-				isDone: true,
-				id: 1
-			},
-			{
-				value: 'Сходить на работу',
-				isDone: false,
-				id: 2
-
-			},
-			{
-				value: 'Полить цветы',
-				isDone: true,
-				id: 3
-			}
-		],
-
-		count: 3,
-	};
-
-const [items, setItems] = useState(initialState.items);
-const [count, setCount] = useState(initialState.count);
-
-useEffect(() => {console.log('componentDidMount')}, []);
-useEffect(() => {console.log('componentDidUpdate')});	
+import React from 'react';
+import {BrowserRouter as Router, Route, Link} from 'react-router-dom';
+import Todo from '../Todo/Todo';
+import About from '../About/About';
+import Contacts from '../Contacts/Contacts';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
 
-const onClickDone = id => {
-	const newItemList = items.map(item => {
-		const newItem = {...item};
-		if (item.id === id) {
-			newItem.isDone = !item.isDone
-		}
-
-		return newItem;
-	});
-
-	setItems(newItemList);
-};
-
-const onClickDelete = id => {
-        const newItemList = items.filter(item => item.id !== id);
-        setItems(newItemList);
-};
-
-const onClickAdd = value => {
-	const newItemList = [
-		...items,
-		{
-			value,
-			isDone: false,
-			id: count + 1
-		}
-	];
-
-	setItems(newItemList);
-	setCount(count => count + 1);
-
-}
+import styles from './App.module.css';
 
-	return (
-		<div className={styles.wrap}>
-			<h1 className={styles.title}>Список дел на сегодня:</h1>
-			<InputItem onClickAdd={onClickAdd} />
-			<ItemList items={items} onClickDone={onClickDone} onClickDelete={onClickDelete} />
-			<Footer count={1} />
-		</div>);
-	};
+const App = () => (<Router>
+	<div className={styles.wrap}>
+        <MenuList className={styles.sidebar}>
+            <Link to='/' className={styles.link}>
+                <MenuItem>Обо мне</MenuItem></Link>
+            <Link to='/todo' className={styles.link}>
+                <MenuItem>Дела</MenuItem></Link>
+            <Link to='/contacts' className={styles.link}>
+                <MenuItem>Контакты</MenuItem></Link>
+        </MenuList>
+            <div>
+                <Route path='/' exact component={About} />
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 4 tabs.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)